_docs_integrate/requests-via-relationshiptemplates.md

```diff
@@ -27,107 +27,141 @@ required_by:
+You can use the [Connector Setup]({% link _docs_operate/setup-with-docker-compose.md %}) guide if you need help installing the Connectors. Since understanding the process of sending a Request via a RelationshipTemplate requires knowledge about [Requests]({% link _docs_integrate/data-model-overview.md %}#request) in general, you should also take a look at our [Request and Response introduction]({% link _docs_integrate/request-and-response-introduction.md %}).
+{: .notice--info}
+
+On the first Connector, which is referred to as the Sender, you will construct the Request and [create the RelationshipTemplate](#create-the-relationshiptemplate) that contains the Request.
+The second Connector, which is referred to as the Recipient, will [receive the Request by loading the RelationshipTemplate](#receive-the-request-by-loading-the-relationshiptemplate). The Recipient then [responds to the Request](#respond-to-the-request).
+Note that the Sender and the Recipient may or may not have already established a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) at the beginning.
 
-On the first Connector, which we will refer to as the Sender, you will create the Request and the RelationshipTemplate.
-The second Connector, which we will refer to as the Recipient, will receive the RelationshipTemplate and respond to the Request.
+A RelationshipTemplate is generally used to establish a Relationship between two Identities. A Request can be sent in this process of establishing a Relationship.
+Nevertheless, a RelationshipTemplate can also be used to exchange Requests between Identities that have already established a Relationship.
+For more information on how to establish Relationships, refer to the [Establish Relationships]({% link _docs_integrate/establish-relationships.md %}) scenario documentation.
+{: .notice--info}
 
-## Check your Request's validity
+## Check the Request's validity
 
-Firstly, you should [check if your Request is valid]({% link _docs_use-cases/use-case-consumption-check-if-outgoing-request-can-be-created.md %}).
-As an example, we use a Request with just an [AuthenticationRequestItem]({% link _docs_integrate/data-model-overview.md %}#authenticationrequestitem), but you can use any Request you want.
-For an overview of the available [RequestItems]({% link _docs_integrate/data-model-overview.md %}#requestitems), check out our [Request and Response introduction]({% link _docs_integrate/request-and-response-introduction.md %}).
+The Sender wants to construct a Request that it can send to the Recipient by inserting it into a RelationshipTemplate.
+Before the Request is sent via the RelationshipTemplate, the Sender should first check its validity by proceeding as described in the documentation of the [Check if outgoing Request can be created]({% link _docs_use-cases/use-case-consumption-check-if-outgoing-request-can-be-created.md %}) use case.
```

Comment:
We should discuss this in general. There are definitely advantages to doing it the way you suggest, as it can shorten sentences and is less maintenance-intensive if the titles of use cases or scenarios change. However, the current variant would have the advantage that it is clearer that you have to leave the page to get further information on how to proceed and that you already know the exact title of the use case or scenario. 

I would suggest that we postpone this discussion for the time being and then correct it in a separate PR if necessary, because it would also affect other scenarios and use cases.